deploy/scripts/boot.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -o errexit
+
```

Comment:
add some checks before running installation that our minimum requirements are matched 
OS, Arch, Available memory 

im not fan of sudo command, when use the boot.sh, the caller should provide relevant permission to run the script as administator you can add it to the requirements 
```
if (( $EUID != 0 )); then
    echo "Please run as root"
    exit
fi
```